scripts/exregional_run_fcst.sh

```diff
@@ -345,8 +345,8 @@ for (( i=0; i<${num_symlinks}; i++ )); do
 
 done
 
-ln_vrfy -sf ${relative_link_flag} ${FIXam}/optics_??.dat ${run_dir}
-ln_vrfy -sf ${relative_link_flag} ${FIXam}/aeroclim.m??.nc ${run_dir}
+#ln_vrfy -sf ${relative_link_flag} ${FIXam}/optics_??.dat ${run_dir}
+#ln_vrfy -sf ${relative_link_flag} ${FIXam}/aeroclim.m??.nc ${run_dir}
 #
 #-----------------------------------------------------------------------
```

Comment:
Why were these commented out?

Reply:
Because they don't work and the loop below (inside `IF ${USE_MERRA_CLIMO}...`) already does what needs to be done.  @willmayfield didn't realize at the time that setting `USE_MERRA_CLIMO="TRUE"` already does what he's trying to do.  Actually, I think it's best to just remove these lines to avoid confusion.

Reply:
Thanks!